src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -185,6 +189,17 @@ export default {
 				'EXP-CORE-854-Dec2022'
 			);
 		}
+
+		const loanCardExpData = getExperimentSettingCached(this.apollo, LOAN_CARD_EXP_KEY);
```

Comment:
This wasn't called out in the ticket, but we also need to use this experiment to swap cards. If you want this to be a separate ticket, please create, but we don't currently have a separate ticket for using this new experiment.

Reply:
I can include on this.
Adding an enable param and passing to children to swap cards for testing group, right?

Reply:
Yup, similar to the swapping we do in LBC based on the other exp flag. It will mean pulling more data in `LoanFinding.vue`, since the controller-based loan card needs all of the data, not just ID. If this effort ends up taking more time, we could instead focus on finishing the new cards, but one of these efforts (experiment or new loan card) needs to be completed this week. We used to use the controller-based cards in lending-home, so at the very least we have code in git history to check out for where we pulled the extra loan data before we introduced the new loan cards recently.

Reply:
Section were not using card controller, is an easy fix now.

Reply:
Oh right, we were using the old classic card, awesome!